install.sh

```
@@ -2174,7 +2455,31 @@ validate_audio_device() {
 
     # Test audio device access - using LC_ALL=C to force English output
     if ! LC_ALL=C arecord -c 1 -f S16_LE -r 48000 -d 1 -D "$device" /dev/null 2>/dev/null; then
-        send_telemetry_event "error" "Audio device validation failed" "error" "step=validate_audio_device,device=$device"
+        # Collect detailed audio device diagnostics
+        local arecord_error=$(LC_ALL=C arecord -c 1 -f S16_LE -r 48000 -d 1 -D "$device" /dev/null 2>&1 | sed 's/"/\\"/g')
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)